xchange-stream-coinbasepro/src/main/java/info/bitrich/xchangestream/coinbasepro/dto/CoinbaseProOrderBuilder.java

```diff
@@ -0,0 +1,207 @@
+package info.bitrich.xchangestream.coinbasepro.dto;
+
+import org.knowm.xchange.coinbasepro.dto.trade.CoinbaseProOrder;
+
+import java.math.BigDecimal;
+
+public class CoinbaseProOrderBuilder {
+    private String id;
+    private BigDecimal price;
+    private BigDecimal size;
+    private String productId;
+    private String side;
+    private String createdAt;
+    private String doneAt;
+    private BigDecimal filledSize;
+    private BigDecimal fillFees;
+    private String status;
+    private boolean settled;
+    private String type;
+    private String doneReason;
+    private BigDecimal executedValue;
+    private String stop;
+    private BigDecimal stopPrice;
+
+    public static CoinbaseProOrder from(CoinbaseProWebSocketTransaction t) {
```

Comment:
"activate" message could also be returned in case we sent a Stop trigger (e.g stoploss order).

Reply:
added